packages/consumption/package.json

```diff
@@ -24,17 +24,27 @@
         "lint:tsc": "tsc --noEmit && tsc -p test/tsconfig.json --noEmit",
         "test:ci": "npm run test",
         "test:ci:lokijs": "USE_LOKIJS=true jest -i",
-        "test:ci:mongodb": "jest -i",
+        "test:ci:mongodb": "jest -i  --coverage=true",
         "test:local:ferretdb": "npm run test:local:start:ferretdb && CONNECTION_STRING='mongodb://localhost:27022' jest",
         "test:local:lokijs": "USE_LOKIJS=true jest",
         "test:local:mongodb": "npm run test:local:start:mongodb && CONNECTION_STRING='mongodb://root:example@localhost:27021' jest",
+        "test:local:coverage": "npm run test:local:lokijs -- --coverage=true",
```

Comment:
IMO we don't need the local scripts. What if you want to test the coverage against mongodb? just run `npm run test:local:mongodb -- --coverage` same for **all** other places.

Reply:
I don't like it either that there are so many npm commands.

I also think we should remove it and maybe document it in the dev README

Reply:
> What if you want to test the coverage against mongodb?

Does this matter?
I can change it, I simply copied it from the connector ;)

Reply:
yeah, it does.